### ISSUE TITLE
Rule proposal：img-has-alt

### ISSUE THREAD

**Issue by snowbabykang** [vuejs/eslint-plugin-vue#1473]
**Please describe what the rule should do:**

This rule should make the “img” tag have an Alt attribute and a value

**What category should the rule belong to?**

[X] Enforces code style (layout)
[ ] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<img src="foo" />
<img src="foo" alt />
<img src="foo" alt="" />
```

**Additional context**

Our company uses Sonar to monitor code quality，sonar always tells us that the "img" tag must have an Alt attribute
![image](https://user-images.githubusercontent.com/12569179/114136811-e345f680-993d-11eb-99ef-9fc5f7240cfb.png)
![image](https://user-images.githubusercontent.com/12569179/114136810-e345f680-993d-11eb-8938-a6086e92786e.png)

**Comment by ota-meshi** [comment]
Thank you for posting this issue.
I think you can use eslint-plugin-vuejs-accessibility.

https://github.com/vue-a11y/eslint-plugin-vuejs-accessibility/blob/master/docs/alt-text.md

**Comment by ota-meshi** [comment]
I close this issue because eslint-plugin-vuejs-accessibility has that rule.
